client/components/Navbar.jsx

```diff
@@ -1,64 +1,114 @@
-// Temporary NavBar: to be changed
-// reference: https://ansonlowzf.com/how-to-build-a-material-ui-navbar/
-
-import * as React from 'react';
-import { AppBar, Toolbar, IconButton, List, ListItem, ListItemText, Container } from '@material-ui/core';
-import Home from '@material-ui/icons/Home';
-import { makeStyles } from '@material-ui/core/styles';
-import { Link } from 'react-router-dom';
+import React, { useState } from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import IconButton from '@material-ui/core/IconButton';
+import Toolbar from '@material-ui/core/Toolbar';
+import Drawer from '@material-ui/core/Drawer';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import clsx from 'clsx';
+import { Link } from '@material-ui/core';
+import { Navlist } from './Navlist';
+import { useStyles } from '../styles/navbarStyles';
 import { useAuth } from '../store/users/AuthContext';
 
-const useStyles = makeStyles({
-  navbarDisplayFlex: {
-    display: 'flex',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-  },
-  navDisplayFlex: {
-    display: 'flex',
-    justifyContent: 'space-between',
-  },
-  linkText: {
-    textDecoration: 'none',
-    textTransform: 'uppercase',
-    color: 'white',
-  },
-});
-
 export const Navbar = () => {
   const classes = useStyles();
+  const [open, setOpen] = useState(false);
   const { currentUser } = useAuth();
 
-  // navbar will changed depending on the loggedStatus of user
-  const loggedStatus = { title: 'login', path: '/login' };
+  // handle opening and closing of app drawer
+  const drawerOpen = () => {
+    setOpen(true);
+  };
 
-  const navLinks = [{ title: 'browse', path: '/browse' }, { title: 'about', path: '/about' }, loggedStatus];
+  const drawerClose = () => {
+    setOpen(false);
+  };
 
-  if (currentUser) {
-    loggedStatus.title = 'logout';
-    loggedStatus.path = '/logout';
-
-    navLinks.unshift({ title: 'create', path: '/quiz/create' });
-  }
+  const links = [
+    { name: 'Browse', route: '/browse', class: 'fas fa-binoculars' },
+    { name: 'Create', route: '/quiz/create', class: 'fas fa-file-alt' },
+    { name: 'Profile', route: '/', class: 'fas fa-user-circle' },
```

Comment:
Should we have the root path `/` be browse instead of profile?

Reply:
that sounds good. I changed it